src/components/AddressAvatar.tsx

```diff
@@ -29,11 +25,11 @@ const AddressAvatar = forwardRef<HTMLDivElement, AddressAvatarProps>(
         )}
         style={{ backgroundColor }}
       >
-        <div className='relative h-full w-full p-[7.5%]'>
+        <div className='relative h-full w-full p-0'>
           <div className='relative h-full w-full'>
             <Image
               sizes='5rem'
-              className='relative'
+              className='relative rounded-full'
```

Comment:
I think this is not needed, because the parent div already has rounded-full and overflow-hidden

Reply:
Yeah, it is not needed now anymore. It'll still be needed though because if an avatar is passed from web3Auth (it doesn't have rounded borders out-of-the-box). I'll remove this as it's not really needed now

Reply:
even for the web3auth avatar, it should be no problem, because the parent already have this
`rounded-full overflow-hidden`, so the parent is already circle, which will clip the image either way

Reply:
that's true, because the corners will not show in the overflow-hidden parent so that rounds the image too...